.changes/v3.6.0/801-improvements.md

```diff
@@ -0,0 +1,6 @@
+* `resource/vcd_network_routed_v2` and `datasource/vcd_network_routed_v2` support VDC groups by
+  inheriting parent VDC or VDC Group from Edge Gateway  [GH-801]
+* `resource/vcd_network_isolated_v2` and `datasource/vcd_network_isolated_v2` support VDC groups via
+  new field `owner_id` replacing `vdc` [GH-801]
+* `resource/vcd_nsxt_network_imported` and `datasource/vcd_nsxt_network_imported` support VDC groups
```

Comment:
Let's capitalize `VDC Groups` across the change log (like you have in docs), so it stands out when scanning with the eyes.